dynamics/client.py

```diff
@@ -474,6 +493,7 @@ def patch(self, data: Dict[str, Any], *, query: Optional[str] = None) -> Dict[st
             data=json.dumps(data).encode(),
             headers={**self.default_headers("patch"), **self.headers},
         )
+        response.raise_for_status()
```

Comment:
See [this comment](https://github.com/MrThearMan/dynamics-client/pull/4/files#r1148326348).